package.json

```diff
@@ -94,10 +96,13 @@
     "clipboard-polyfill": "^3.0.1",
     "d3": "^5.16.0",
     "date-fns": "^2.16.1",
+    "detect-browser": "^5.2.0",
     "elliptic": "^6.5.3",
     "eth-contract-metadata": "^1.15.0",
     "ethereum-blockies": "^0.1.1",
+    "ethjs": "^0.4.0",
```

Comment:
cc @guanbinrui do we need so many eth lib in our app?

Reply:
removed